allauth/socialaccount/providers/amazon_cognito/provider.py

```diff
@@ -57,7 +52,10 @@ def extract_extra_data(self, data):
         return {
             "address": data.get("address"),
             "birthdate": data.get("birthdate"),
+            "email": data.get("email"),
```

Comment:
This method was not introduced in this PR, but it does not make any sense. I think we need to replace it with just this:

```
    def extract_extra_data(self, data):
        ret = dict(data)
        phone_number_verified = data.get("phone_number_verified")
        if phone_number_verified is not None:
            ret["phone_number_verified"] = (
                convert_to_python_bool_if_value_is_json_string_bool(
                    "phone_number_verified"
                )
            )
        return ret
```

Reply:
Done.